opensrp-chw/src/kk/assets/json.form/family_member_register.json

```diff
@@ -73,6 +73,47 @@
           "err": "{{family_member_register.step1.unique_id.v_required.err}}"
         }
       },
+      {
+        "key": "primary_caregiver_consent",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "",
+        "type": "check_box",
+        "label": "{{family_member_register.step1.primary_caregiver_consent.label}}",
+        "exclusive": [
+          "intervention_consent_yes",
+          "intervention_consent_no"
+        ],
+        "options": [
+          {
+            "key": "intervention_consent_yes",
```

Comment:
@nkafwimi for the primary caregiver consent should the fieldname be the same like this?

Reply:
@issyzac I see that on the data dictionary it was specified as: primary_caregiver_consent
![Uploading Screen Shot 2023-01-30 at 15.00.32.png…]()

Reply:
@nkafwimi yes, thats the field name, I meant to ask for the value of that field, but I think its not really a big of a concern since the field name already specifies that its for the primary caregiver and also they are consenting to be in the intervention